### PR TITLE
Add user workflow for documenting bugs

### LINKED ISSUE #509: User workflow for documenting bugs.
Users who may be non-coders need to be able to report bugs they come across while using the editor.

The general "report a bug" option on the publiclab.org footer is not fully optimized for reporting bugs from the editor, as documenting the editor bugs may require more than text description.
 Including screenshots, GIFs and screen recordings will make understanding and reproducing the bugs a lot easier.

### PR DESCRIPTION
**Pull request by Shulammite-Aso**
Fixes #509 
The button before the ple-footer is a link that takes a user to the editor-bug-report template here https://publiclab.org/post?n=23853&title=Give%20this%20bug%20a%20title%20here

Screenshot from 2020-06-15 19-34-42

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `grunt jasmine`
* [ ] code is in uniquely-named feature branch and has no merge conflicts
* [ ] PR is descriptively titled
* [ ] PR body includes `fixes #0000`-style reference to original issue #
* [ ] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. 

Thanks!

### CODE REVIEW

**Comment by Shulammite-Aso** [comment]
@jywarren @emilyashley @shreyaa-sharmaa @keshav234156 @NitinBhasneria please take a look at this.

File: dist/PublicLab.Editor.js [hunk 1]
```diff
@@ -21778,6 +21779,10 @@ module.exports = PublicLab.MainImageModule = PublicLab.Module.extend({
       progressall: function (e, data) {
 
         var progress = parseInt(data.loaded / data.total * 100, 10);
+
```
Comment (jywarren) [inline dist/PublicLab.Editor.js:21782]: Hmm, maybe this needs to be rebuilt without unrelated changes?

File: examples/index.html [hunk 2]
```diff
@@ -302,6 +302,9 @@ <h4><b>By entering co-ordinates :</b></h4>
         </div>
         <!-- end multi module -->
       </div>
+      <div class="bug-btn-wrapper">
+        <a href="https://publiclab.org/post?n=23853&title=Give%20this%20bug%20a%20title%20here" class="bug-btn" target="_blank">Found an issue with the editor? Report a bug.</a>
```
Comment (jywarren) [inline examples/index.html:306]: Here, could we make the title say either nothing (user can input their own) or "Editor bug report: " and they can add their own after that?

File: dist/PublicLab.Editor.js (hunk 1 above)
Comment (Shulammite-Aso) [reply dist/PublicLab.Editor.js]: I don't know how these changes go added here, but I have just resolved it.

**Comment by Shulammite-Aso** [comment]
Have made the changes @jywarren

**Comment by govindgoel** [comment]
Looks Fine!
@jywarren please review this

**Comment by jywarren** [comment]
This looks perfect. Thank you so much @Shulammite-Aso !!!

Just checking if you saw this post on the release schedule: https://github.com/publiclab/PublicLab.Editor/issues/532#issuecomment-644399923

just want to be sure you're in sync, thank you!

**Comment by jywarren** [comment]
Done! Great! So, i think we need to be sure this gets reflected not just in the example but in the `plots2` usage as well. The corresponding file in the `plots2` integration is here:

https://github.com/publiclab/plots2/blob/master/app/views/editor/rich.html.erb

Can you open a new PR on `plots2` making the same change (probably the CSS won't be needed if we ensure that a new release of the Editor is merged before this change is added? Or we can just redundantly add the CSS change to `plots2` but leave a comment that it can be removed after the next version of the Editor. 

Thank you @Shulammite-Aso ! It'll be great to see this in production!

**Comment by Shulammite-Aso** [comment]
> Done! Great! So, i think we need to be sure this gets reflected not just in the example but in the `plots2` usage as well. The corresponding file in the `plots2` integration is here:
> 
> https://github.com/publiclab/plots2/blob/master/app/views/editor/rich.html.erb
> 
> Can you open a new PR on `plots2` making the same change (probably the CSS won't be needed if we ensure that a new release of the Editor is merged before this change is added? Or we can just redundantly add the CSS change to `plots2` but leave a comment that it can be removed after the next version of the Editor.
> 
> Thank you @Shulammite-Aso ! It'll be great to see this in production!

Oh yeah, will certainly do this.
Thanks!!
